shared/config.default/custom_auth/test.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
There is already a basic example for custom_auth. I'd rather not add more just for testing.